doc/xplat_cli_migration.md

```diff
@@ -0,0 +1,223 @@
+* Scripts are not compatible between both CLIs
+
+## Why consider trying the Azure CLI 2.0?
+Regardless of the tool or vendor, it is important to ensure you benefit when
+adding a new tool to your toolbox.  While the following covers only a small
+number of difference between these two products, we expect customers to benefit
+from the following:
+
+* Clean outputs for common workflows
+  * `--out table` for simplified human output
+  * `--out json` and `--out jsonc` for JSON outputs
+  * `--out tsv` for interacting with tools like grep, AWK, and jq
+* Improved and consistent in-tool documentation with `-h`
+  * Includes descriptions for groups, commands, and parameters
+  * Indicates where to look up required values
+  * Examples and links to web content
+* Improved command-line productivity
+  * Use `[tab][tab]` to lookup parameters, including resource groups and names (only supported in BASH and BASH on Windows)
+  * Work with either ID values (`--ids`) _or_ resource group and name (`-g -n`)
+  * Built in client-side query engine powered by JMESPath
+* (Coming) Support for more features and services
+  * Our preview meets or exceeds XPlat CLI functionality for Compute, Storage, Network, RBAC, and ARM
+  * More services coming online soon!
+
+While we believe the above list is compelling, it's important to remember
+**the Azure CLI 2.0 only supports ARM mode**.  You must continue to use
+the Azure XPlat CLI for ASM/Classic mode management.
+
+## Getting both CLI's setup side-by-side
+
+First, you run `azure --version` and ensure you are using `0.10.5` or later, as
+this is required for sharing your credentials between both CLIs.  If you installed
+using NPM, upgrade with `npm upgrade -g azure-cli`.  If you used an installer,
+we recommend downloading the latest installer to upgrade.
+
+To install the Azure CLI 2.0, follow the steps for your preferred platform or
+environment on our [Installation Guide](https://github.com/Azure/azure-cli/blob/master/doc/preview_install_guide.md).
+
+Once installed, you can run `az configure` and follow the steps to setup your default output format.  
+
+Then run `az login` to login using device authentication.  Once this step is complete you should be authenticated to use both CLIs.  
+
+## Important new concepts in the Azure CLI 2.0
+Here is a quick list of some new and changed concepts that can help you understand the new tool.
+
+* Interactive Concepts
+  * Use `az configure` to setup your default output format
+  * You will find help to be generally more useful, try `az vm create -h` for an example
+  * Positional parameters are not supported, use `az vm list -g MyGroup` instead of `azure vm list MyGroup`
+* Automation and Scripting Concepts
+  * You can refer to resources using their full ID URI with `--ids` or with the resource group and name using `-g [rg] -n [name]`
+  * Use `--query "[expression]"` to extract single values
+  * Use `--out tsv` when getting a single value to prevent any extra markup
```

Comment:
>to prevent any extra markup [](start = 48, length = 27)

it seems like this could be worded more positively :), "to get plain value output" or something